### ISSUE TITLE
Is calculation of Co incorrect?

### ISSUE THREAD

**Issue by kevinwolz** [kevinwolz/sysdesign#18]
Regarding the calculation of Co, the non-centrality percentage, Parrot et al. (2012) and others mention that this number should be limited to 5%, "which equates to an alpha of 1.1". However, I don't see how the provided equation for Co leads to that equivalence. When I plug an alpha of 1.1 into Parrot Equation 2, I get a Co of -2.38, not something around 5. What am I missing here??

I've emailed the three authors of Parrot et al. (2012) this question.

**Comment by kevinwolz** [comment]
Received a response from John Lohtka. The confirmed that the calculation is correct. They think the discrepancy just has to do with rounding that Nelder must have done. Here is his full response:

_Kevin,
Thank you for your question. I have discussed your question with my co-authors and we want to provide you some feedback.

In our paper we did not delve into this issue, because after we present this equation and limit, we immediately cite the Namkoong (1966) rebuttal to this limitation when dealing with trees (This deviation would have negligible impact on trees, unlike row crops).  However, I think the 1.1 limitation Nelder presents is a general guide: "this implies a value of a less than about 1.1, a limit which has not proved unduly restrictive in practice."  While alpha of 1.1 has a Co of only 2.38%, if you were to have an alpha of 1.2, you have a Co of 4.85%.  We cannot speak for Nelder, but looking at the numbers we think the 1.1 limit was given based on having alpha with single decimal place and rounding numbers up?  Assuming this, Nelder saw the 1.2 alpha = Co 5% and 1.1 alpha was just the next level down. Regardless, this limitation has been shown unnecessary in tree growth research (there have been studies with much higher non-centrality using these designs).

Short version: The equation is correct. The alpha limitation of 1.1 is not important in tree research and is not to be considered equivalent to the 5% Co.  It seems to be just a general guideline Nelder presented. 

Hope these thoughts help you with the work on your R package.

Best regards,
John

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
John M. Lhotka, Ph.D.
Associate Professor of Silviculture
University of Kentucky - Department of Forestry
210 T.P. Cooper Building Lexington, KY 40546-0073_
